src/map/clif.h

```diff
@@ -162,6 +162,13 @@ enum e_party_invite_reply {
 	PARTY_REPLY_INVALID_MAPPROPERTY_ME, ///< return=9 : !TODO "Cannot join a party in this map" -> MsgStringTable[1871] (since 20110205)
 };
 
+typedef enum PARTY_MEMBER_WITHDRAW {
```

Comment:
Could you move it to mmo.h(so it can be used in the inter server too) and remove the typedef?